lib/platform.js

```diff
@@ -1,21 +0,0 @@
-let crypto = require('crypto');
```

Comment:
This one was first moved in 8483c05dcf82e552a29ac569812a12f7e63cba81 and then changed, however git doesn't show it as moved anymore due to the extension + content change.